designer/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/EspItTest.scala

```diff
@@ -173,24 +173,24 @@ trait EspItTest extends LazyLogging with WithHsqlDbTesting with TestPermissions
       updateProcess(processName, process)(testCode)
     }
 
-  protected def saveProcess(process: DisplayableProcess)(testCode: => Assertion): Assertion =
-    createProcessRequest(ProcessName(process.id)) { code =>
+  protected def saveProcess(process: DisplayableProcess, category: ProcessingType)(testCode: => Assertion): Assertion =
```

Comment:
I think it would be simpler to have process.category.get here and assume that in tests we have category in displayable

Reply:
ok